src/main/java/org/galatea/starter/domain/IexHistoricalPrice.java

```diff
@@ -19,4 +21,23 @@
   private BigDecimal open;
   private BigDecimal volume;
   private String date;
+
+  public static List<IexHistoricalPrice> createFromHistoricalEntityList(
+      List<IexHistoricalPriceEntity> historicalEntities) {
+    List<IexHistoricalPrice> historicalPrices =
+        new ArrayList<>();
+
+    for (IexHistoricalPriceEntity entity : historicalEntities) {
```

Comment:
If you're interested, here is a link to my presentation on Java Streams
https://docs.google.com/presentation/d/1SKWAiVWqcpwxFWu97BDYspc37Nu0FhfPV_-3-q6E_qc/edit#slide=id.p
Even though everything you can do with Java Streams is possible without Java Streams, Java Streams can make some operations much easier / fewer lines